eng/targets/Imports.targets

```diff
@@ -14,7 +14,7 @@
 
     <!-- Only generate our runtimeconfig.json files for net core apps. It's unnecessary in desktop projects
          but gets included in lots of output items like VSIX. -->
-    <GenerateRuntimeConfigurationFiles Condition="'$(TargetFramework)' != 'netcoreapp1.1' AND '$(TargetFramework)' != 'netcoreapp2.1'">false</GenerateRuntimeConfigurationFiles>
+    <GenerateRuntimeConfigurationFiles Condition="'$(TargetFramework)' != 'netcoreapp1.1' AND '$(TargetFramework)' != 'netcoreapp2.1' AND '$(TargetFramework)' != 'netcoreapp3.0'">false</GenerateRuntimeConfigurationFiles>
```

Comment:
Ideally you should just have a condition based on `TargetFrameworkIdentifier`.  However, that only will work if this targets file is evaluated after `Microsoft.NET.TargetFrameworkInference.targets`, so I don't know if that's an option here.

Reply:
@dsplaisted this is hooked during Directory.Build.targets. Is that in the right place?

Reply:
@jaredpar Yes, you should just be able to replace this condition with `'$(TargetFrameworkIdentifier)' != '.NETCoreApp'`.  On the other hand, as Nick mentioned in Slack, the whole condition is probably working around an old bug in the SDK and you can probably just delete the line entirely.